Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -464,8 +464,8 @@ namespace pixelTopology {
 
     static constexpr float dzdrFact = 8 * 0.0285 / 0.015;  // from dz/dr to "DY"
 
-    static constexpr int minYsizeB1 = 36;
-    static constexpr int minYsizeB2 = 28;
+    static constexpr int minYsizeB1 = 1;
+    static constexpr int minYsizeB2 = 1;
```

Comment:
could these parameters be exposed as configurable parameters instead of hardcoded? 
I guess this would ease the checks to be asked to various stakeholders.

Reply:
Should be done in ef9d50d261216f919a1442bea90684e299925e7e

Reply:
thanks!